Servers/UI/OJS.Servers.Ui/ClientApp/src/components/file-uploader/FileUploader.tsx

```diff
@@ -42,14 +45,23 @@ const FileUploader = ({ file, problemId }: IFileUploaderProps) => {
                 return;
             }
 
+            const uploadedFile = eventTarget[0];
+            const extension = uploadedFile.name.split('.').pop();
+
+            if (allowedFileExtensions && !allowedFileExtensions.includes(extension) && !isNil(onInvalidFileExtension)) {
```

Comment:
Is the check !isNil(onInvalidFileExtension) really needed ? The property is not marked as nullable, so there should be no chance to receive null there.